Engine/Component/ComponentBillboard.cpp

```diff
@@ -0,0 +1,235 @@
+#include "ComponentBillboard.h"
+
+#include "Main/Application.h"
+#include "Module/ModuleProgram.h"
+#include "Module/ModuleResourceManager.h"
+#include "Module/ModuleRender.h"
+#include "Module/ModuleTime.h"
+
+#include "ResourceManagement/ResourcesDB/CoreResources.h"
+
+ComponentBillboard::ComponentBillboard() : Component(nullptr, ComponentType::BILLBOARD)
+{
+	InitData();
+}
+
+ComponentBillboard::ComponentBillboard(GameObject * _owner) : Component(owner, ComponentType::BILLBOARD)
```

Comment:
Here you are not using the owner sent and you have code style.

Reply:
done